examples/react-todos-app/code-pushup.config.js

```diff
@@ -24,9 +29,37 @@ export default {
       eslintrc: '.eslintrc.js',
       patterns: ['src/**/*.js', 'src/**/*.jsx'],
     }),
+    await lighthousePlugin('https://codepushup.dev/', {
+      outputPath: join('.code-pushup', LIGHTHOUSE_REPORT_NAME),
+    }),
   ],
   categories: [
-    // TODO: add performance category once Lighthouse plugin implemented, include eslintAuditRef('react-jsx-key', 0)
+    {
+      slug: 'performance',
+      title: 'Performance',
+      refs: [lighthouseGroupRef('performance')],
+    },
+    {
+      slug: 'a11y',
+      title: 'Accessibility',
+      refs: [lighthouseGroupRef('accessibility')],
+    },
+    {
+      slug: 'best-practices',
+      title: 'Best Practices',
+      refs: [lighthouseGroupRef('best-practices')],
+    },
+    {
+      slug: 'seo',
+      title: 'SEO',
+      refs: [lighthouseGroupRef('seo')],
+    },
+    {
+      slug: 'pwa',
+      title: 'PWA',
+      isBinary: true,
+      refs: [lighthouseGroupRef('pwa'), eslintAuditRef('react-jsx-key', 1)],
+    },
```

Comment:
The lint rule is supposed to be in _Performance_ category, not _PWA_:

```suggestion
    {
      slug: 'performance',
      title: 'Performance',
      refs: [lighthouseGroupRef('performance'), eslintAuditRef('react-jsx-key', 1)],
    },
    {
      slug: 'a11y',
      title: 'Accessibility',
      refs: [lighthouseGroupRef('accessibility')],
    },
    {
      slug: 'best-practices',
      title: 'Best Practices',
      refs: [lighthouseGroupRef('best-practices')],
    },
    {
      slug: 'seo',
      title: 'SEO',
      refs: [lighthouseGroupRef('seo')],
    },
    {
      slug: 'pwa',
      title: 'PWA',
      isBinary: true,
      refs: [lighthouseGroupRef('pwa')],
    },
```

Reply:
removed